pyproject.toml

```diff
@@ -25,6 +25,7 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

Comment:
Technically listing these is probably not that helpful, since it will support all future Python 3 versions. You could just list `"Programming Language :: Python :: 3.12"`. But listing what you test on is fine too.